src/webui/prefjson.cpp

```diff
@@ -114,6 +114,7 @@ QByteArray prefjson::getPreferences()
     data["ip_filter_enabled"] = session->isIPFilteringEnabled();
     data["ip_filter_path"] = Utils::Fs::toNativePath(session->IPFilterFile());
     data["ip_filter_trackers"] = session->isTrackerFilteringEnabled();
+    data["banned_IPs"] = session->bannedIPs().join("\n");
```

Comment:
@qbittorrent/qbittorrent-frequent-contributors should we join the multiple IPs(ipv4+ipv6) using newlines or some other delimiter like `|`? Do we have any informal convention on this?